addon/components/power-calendar/days.js

```diff
@@ -18,6 +18,16 @@ function withLocale(locale, fn) {
   return returnValue;
 }
 
+const WEEK_DAYS = [
```

Comment:
how about `Mon_Tue_Wed_Thu_Fri_Sat_Sun.split('_')` for visibility and readability?